test/mmio_bus.cpp

```diff
@@ -37,93 +37,101 @@ using bus = groov::mmio_bus<iface>;
 
 TEST_CASE("fully and partially write a 32-bit without reading it",
           "[mmio_bus]") {
-    reg32 = 0x87654321u;
+    reg32 = 0x8765'4321u;
     iface::num_stores = {};
     iface::num_loads = {};
 
     SECTION("write the whole register") {
-        bus::write<0xffffffffu>(addr32, 0xc001d00du) | async::sync_wait();
+        bus::write<0xffff'ffffu, 0u, 0u>(addr32, 0xc001'd00du) |
+            async::sync_wait();
 
         CHECK(iface::num_stores == 1);
         CHECK(iface::num_loads == 0);
-        CHECK(reg32 == 0xc001d00du);
+        CHECK(reg32 == 0xc001'd00du);
     }
 
     SECTION("write byte 0") {
-        bus::write<0xffu>(addr32, 0x38u) | async::sync_wait();
+        bus::write<0xffu, 0xffff'ff00u, 0u>(addr32, 0x38u) | async::sync_wait();
```

Comment:
issue: i believe the identity mask should be `0u`, right? this test should only be testing the subword write optimization, not the identity optimization. unless i have my understanding of the identity mask bit values inverted.

Reply:
Yes - reverted to `0u`.